src/components/WebcamCapture/WebcamCapture.js

```diff
@@ -1,13 +1,18 @@
 import React, {Component} from 'react';
-import Webcam from 'react-webcam';
-import Notification from './../Notification/Notification';
 import PropTypes from 'prop-types';
+
+import Webcam from 'react-webcam';
+import Notification from '../Notification/Notification';
+import ViewFinder from './ViewFinder';
+
 import './WebcamCapture.css';
 
 class WebcamCapture extends Component {
   state = {
     isDetecting: true,
-    cameraConnected: false
+    cameraConnected: false,
+    successTime: null,
```

Comment:
what is this used for?

Reply:
Oops, left that in by mistake, removed it now.